src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const tr = document.querySelectorAll('tr');
+
+for (const i of tr) {
```

Comment:
Try to use more meaningful variable naming. Your colleagues may not understand what they mean `i` and `tr`